bucket/deew.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "3.0.1",
+    "description": "Dolby Encoding Engine Wrapper",
+    "homepage": "https://github.com/pcroland/deew",
+    "license": "MIT License",
+    "suggest": {
+        "ffmpeg": [
+            "ffmpeg",
+            "versions/ffmpeg-nightly"
+        ]

```

```suggestion
        "ffmpeg": "ffmpeg"
```

Suggests stable package